src/app/pages/signing/signing.component.ts

```diff
@@ -61,8 +61,11 @@ export class SigningComponent implements OnInit {
 
     // remove navigation popup
     this.sending = true;
-    this.msgIo.sendMessage(newMsg, this.key).then(() =>
-      this.router.navigate(['/msg-sent'], { queryParams: { name: this.name } }))
-      .catch(error => alert(error));
+    this.msgIo.sendMessage(newMsg, this.key).then(() => {
+      this.router.navigate([ '/msg-sent' ], { queryParams: { name: this.name } });
+    }).catch(error => {
+        alert(error);
```

Comment:
Use `AuthService.displayError()`
```suggestion
        AuthService.displayError(error);
```